linode/vpcsubnet/resource_test.go

```diff
@@ -139,6 +141,53 @@ func TestAccResourceVPCSubnet_update(t *testing.T) {
 	})
 }
 
+func TestAccResourceVPCSubnet_create_InvalidLabel_basic(t *testing.T) {
+	t.Parallel()
+
+	subnetLabel := acctest.RandomWithPrefix("tf-test")
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:                 func() { acceptance.PreCheck(t) },
+		ProtoV5ProviderFactories: acceptance.ProtoV5ProviderFactories,
+		CheckDestroy:             checkVPCSubnetDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config:      tmpl.Basic(t, vpcID, subnetLabel, "172.16.0.0/24"),
```

Comment:
Looks like the subnet here conflicts with the subnet created in `TestAccResourceVPCSubnet_basic`. We might need to either make these unique or create per-test VPCs 👍